File: Python/002.py
def fibsum(bound):
  sum = 0
  a, b = 0,1

  while (b < bound):
    a, b = b, a + b 
    if (b % 2 == 0):
      sum += b

  return sum 

print "The fibsum is: ", fibsum(4000000)

File: Javascript/002.js
module.exports = {
  'run': function() {
    var curr, next, prev, sum;
    sum = 0;
    prev = 1;
    curr = 2;
    next = 0;
    while (curr < 4000000) {
      if (curr % 2 === 0) {
        sum += curr;
      }
      next = prev + curr;
      prev = curr;
      curr = next;
    }
    console.log('The sum is: ' + sum);
  }
};
